concordia/templates/transcriptions/asset_detail.html

```diff
@@ -183,7 +183,7 @@ <h5 class="mx-half offwhite-text" id="register-login-reminder">Only registered u
             prefixUrl: '{% static "vendor/openseadragon/images/" %}',
             tileSources: {
                 type: 'image',
-                url: "{% asset_media_url asset %}"
+                url: "{% if campaign.s3_storage %}{% asset_s3_url asset %}{% else %}{% asset_media_url asset %}{% endif %}"
```

Comment:
This should be removed and replaced with setting MEDIA_URL in the settings file to whatever the public endpoint is, such as the public URL for an S3 bucket, CloudFront, or CloudFlare.